modules/sildisco/src/Auth/Process/AddIdp2NameId.php

```diff
@@ -32,8 +33,6 @@ class AddIdp2NameId extends ProcessingFilter
 
     const SP_NAMEID_ATTR = 'saml:sp:NameID'; // The key for the NameID
 
-    const VALUE_KEY = 'Value';  // The value key for the NamedID entry
```

Comment:
Wasn't used anywhere?

Reply:
No, but I realize that it could be used outside this repo. I'll put it back...reluctantly.